Core/include/Acts/Surfaces/CylinderSurface.hpp

```diff
@@ -49,7 +49,7 @@ class CylinderSurface : public Surface {
   /// @param halfphi The half opening angle
   /// @param avphi The phi value from which the opening angle spans (both sides)
   CylinderSurface(const Transform3& transform, double radius, double halfz,
-                  double halfphi = M_PI, double avphi = 0.);
+                  double halfphi = M_PI, double avphi = 0., double angle = 0.);
```

Comment:
I might be missing something here, but I would have thought this `angle` parameter could be restricted to `CylinderBounds` (and probably `CylinderVolumeBounds`).

Reply:
Should be gone now